docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,137 @@
+# Kubernetes Deployment
+
+## AnyCable-Go
+
+anycable-go can be easily deployed to your Kubernetes cluster using Helm and [our official Helm chart][anycable-helm].
+
+ 1. Add it as a dependency to your main application:
+
+    ```yaml
+    # Chart.yaml
+    dependencies:
+    - name: anycable-go
+      version: 0.2.4
+      repository: https://helm.anycable.io/
+    ```
+
+    Check latest Helm chart version at [github.com/anycable/anycable-helm/releases](https://github.com/anycable/anycable-helm/releases).
+
+ 1. And then configure it in your application values within `anycable-go` namespace:
+
+    ```yaml
+    # values.yaml
+    anycable-go:
+      env:
+        # Assuming that Ruby RPC is available in K8s in the same namespace as anycable-rpc service (see next chapter)
+        anycableRpcHost: anycable-rpc:50051
+      ingress:
+        enable: true
+        path: /cable
+
+    # values/production.yaml
+    anycable-go:
+      env:
+        # Assuming that Redis is available in K8s in the same namespace as redis-anycable service
+        anycableRedisUrl: redis://:password@redis-anycable:6379/0
```

Comment:
I'm not sure that it's a good practice even as an example. Peeps will be copypasting it.